includes/core/classes/class-event.php

```diff
@@ -350,13 +341,7 @@ public function get_datetime(): array {
 			(array) $data
 		);
 
```

Comment:
update docblock in method and remove things about user settings since we moved it out.